### ISSUE TITLE
Categorical Reparameterization with Gumbel-Softmax

### ISSUE THREAD

**Issue by icoxfog417** [arXivTimes/arXivTimes#45]
## 一言でいうと

離散値の予測(=クラス分類など)を行う際の出力について。クラスの特定に使うargmaxなどは微分可能でないためうまく誤差伝搬できない。なので、連続/離散分布の特性を調整する変数を導入した方法を提案。その名はガンベル・ソフトMAX

### 論文リンク

http://blog.evjang.com/2016/11/tutorial-categorical-variational.html

### 著者/所属機関

Eric Jang, Shixiang Gu, Ben Poole
Google Brain
University of Cambridge
Stanford University

**Comment by icoxfog417** [comment]
Keras実装

https://github.com/EderSantana/gumbel
